automatic/octave.portable/update.ps1

```diff
@@ -0,0 +1,59 @@
+[CmdletBinding()]
+param([switch] $Force)
+
+Import-Module AU
+
+$domain = 'https://www.gnu.org'
+$releases = "${domain}/software/octave/news.html"
+
+function global:au_BeforeUpdate {
+  $checksumType = 'sha256'
+  $Latest.ChecksumType64 = $Latest.ChecksumType32 = $checksumType
+
+  $Latest.Checksum32 = Get-RemoteChecksum $Latest.URL32 -Algorithm $checksumType
+  $Latest.Checksum64 = Get-RemoteChecksum $Latest.URL64 -Algorithm $checksumType
+}
```

Comment:
Since I've asked you to fully embed the package, then make use of the following AU function `Get-RemoteFIles`.